src/style.css

```diff
@@ -0,0 +1,31 @@
+*,
+*::after,
+*::before {
```

Comment:
You don't need to specify `::after` and `::before` here; delete this. Why do you use it here?

Reply:
Yes, sorry in this project I didn't use `::before` and `::after`.